frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -222,35 +222,36 @@ const fetchNodeMetrics = (): Promise<NodeMetrics> => {
   const metrics = [
     {
       key: 'usedMemory',
-      query: 'sum by (instance) (node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes)',
+      query: 'node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes',
     },
     {
       key: 'totalMemory',
-      query: 'sum by (instance) (node_memory_MemTotal_bytes)',
+      query: 'node_memory_MemTotal_bytes',
     },
     {
       key: 'usedStorage',
-      query: 'sum by (instance) (node_filesystem_size_bytes - node_filesystem_free_bytes)',
+      query: 'instance:node_filesystem_usage:sum',
     },
     {
       key: 'totalStorage',
-      query: 'sum by (instance) (node_filesystem_size_bytes)',
+      query:
+        'sum by(instance) (topk(1, node_filesystem_size_bytes{device!="tmpfs",device!="nsfs",mountpoint!="/boot",mountpoint!="/boot/efi"}) by (device, instance))',
```

Comment:
To explain the thinking behind this query. We don't want to count tmpfs, since that is really memory. We don't want to count nsfs because that is all of the mountpoints created for all of the containers and no actual storage value is included for those (they are always 0). We don't want to include /boot and /boot/efi because those are small partitions dedicated to booting the operating system and are storage that you can't really directly use for the cluster. What should be left are any disks in use for the node. The same device will have multiple time series recorded because the same device is mounted at several locations, so the topk makes sure we only grab one time series per device (they will always report the same value). Customers can choose to attach more than one disk, so we do still need a sum. On an OOTB default AWS deployment this means 119.5 GB of the 120GB disk that is attached will show as the total storage available for the node.

cc @eparis who I debated this with

Reply:
The filesystem usage is recorded by cluster monitoring using the method I originally had in here which is to just look at the mountpoint="/" however as Eric pointed out to me customers can attach multiple disks, so just looking at that mountpoint wouldn't get an accurate recording of the filesystem usage in all environments (it would be right in most environments).

Cluster monitoring recording rule:
https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/prometheus-k8s/rules.yaml#L714-L716

Reply:
I guess it means that we need to fix the `instance:node_filesystem_usage:sum` rule to take into account setups with multiple mountpoints per node? I've dug in the [kube-prometheus](https://github.com/coreos/kube-prometheus)'s history from where this rule originates but I couldn't find any issue/PR mentioning this. 
cc @paulfantom @brancz

Reply:
Yes I would also prefer if this was solved in the recording rule, as that way we can make sure everyone who uses the recording rule gets the expected result. Otherwise we end up in chaos all the time, because everything needs to be individually adapted.

Reply:
@brancz agree we should update the recording rule for usage, wdyt about a second recording rule for total?

Reply:
I have 2 issues here

1) If `instance:node_filesystem_usage:sum` is based on `node_filesystem_size - node_filesystem_free` then why do we compute this when node_exporter exposes `node_filesystem_avail` metric (in node_exporter 1.0+ it changed name to `node_filesystem_avail_bytes`)? /cc @brancz @simonpasquier 

2) `instance:node_filesystem_usage:sum` is bugged when using node_exporter 1.0 (used in OpenShift 4.6+) as `node_filesystem_size` and `node_filesystem_free` metrics changed names (source: https://prometheus.demo.do.prometheus.io/graph?g0.range_input=1h&g0.expr=node_filesystem_size_bytes&g0.tab=1). Filed a BZ in https://bugzilla.redhat.com/show_bug.cgi?id=1860016 to fix `instance:node_filesystem_usage:sum`.